app/Platform.php

```diff
@@ -24,6 +24,16 @@ public static function normalizePath(string $path): string
         return $path;
     }
 
+    public static function getGlobalComposerHome(): string
+    {
+        return rtrim(shell_exec('composer -n config --global home'), "\n");
+    }
+
+    public static function getGlobalComposerBinDir(): string
+    {
+        return rtrim(shell_exec('composer -n global config bin-dir --absolute --quiet'), "\n");
```

Comment:
```suggestion
        return rtrim(shell_exec('composer -n config --global bin-dir --absolute --quiet'), "\n");
```

Reply:
the `--global` flag actually behaves differently (at least on my system, lol). We might need to change the other spot I use a global cli command instead

Reply:
This is a weird one. The `composer global home` command works as expected using both methods, but trying the `bin-dir` command in tinkerwell on another project returns unexpected results:

```php
rtrim(shell_exec('composer -n config --global bin-dir --absolute --quiet'), "\n"); // "/Users/gopher/Code/EXACTsports/camps/vendor/bin"
rtrim(shell_exec('composer -n global config bin-dir --absolute --quiet'), "\n"); // "/Users/gopher/.composer/vendor/bin"
```

Reply:
Yeah yeah I just realised that, I have updated it at least keep the order `config - global` same as the one before.

Reply:
playing around in tinkerwell again it looks like it's import that the `global` keyword comes first. I might just change the other command to match if it works the same

Reply:
Yeah `global` should come first in both cases